.travis.yml

```diff
@@ -11,6 +11,7 @@ addons:
       - libpcap-dev
       - libelf-dev
       - gcc-multilib
+      - wireshark
```

Comment:
Travis doesn't actually run the tests yet, and when it does it'll be in a VM (since we need a newer kernel). So no reason to add this here; I meant that you should add it as a BuildRequires in `packaging/rpm/xdp-tools.spec`

Reply:
Will remove, and add wireshark to the spec file.